public/manifest.json

```diff
@@ -1,25 +1,13 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
-    {
-      "src": "logo192.png",
-      "type": "image/png",
-      "sizes": "192x192"
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
-  "start_url": ".",
-  "display": "standalone",
-  "theme_color": "#000000",
-  "background_color": "#ffffff"
-}
+    "short_name": "NER Icon",
+    "name": "NER Club Icon",
+    "icons": [{
+        "src": "NERfavicon.ico",
+        "sizes": "192x192 512x512",
```

Comment:
I think I chose these based on a comment from Kevin, I think he said to use those 2 sizes. There might though be cases where access to a smaller file might be helpful

Reply:
I think his comment means that we need a different png file of the same image to serve for the 192x192 and 512x512 sizes, and the ico file should serve for the smaller sizes. This would make it much more similar to the template manifest.json.